endpoints-support.md

```diff
@@ -12,7 +12,7 @@
 
 * If an endpoint is marked as implemented in a previous version of the API, it will likely already be working for newer API versions, however in these cases it is important to:
 1. Specify the 'type' of the resource when using an untested API, as it will not get set by default
-2. If an example is not working, verify the [HPE OneView REST API Documentation](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview3.1/cic-api/en/api-docs/current/index.html)  for the API version being used, since the expected attributes for that resource might have changed.
+2. If an example is not working, verify the HPE OneView REST API Documentation ([OneView 4.0](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview4.0/cic-api/en/index.html), [OneView 4.1](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview4.1/cic-api/en/index.html), [OneView 4.2](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview4.2/cic-api/en/index.html)) for the respective API version being used, since the expected attributes for that resource might have changed.
```

Comment:
i think we can keep the latest API doc link only which is supported for this provider currently i.e. API 800. otherwise list will keep on growing whenever there is a new API supported.